src/contexts/Maci.tsx

```diff
@@ -52,9 +52,15 @@ export const MaciProvider: React.FC<MaciProviderProps> = ({ children }) => {
 
   const attestationId = useMemo(() => {
     const values = attestations.data?.valueOf() as Attestation[] | undefined;
-    const length = values?.length ?? 1;
 
-    return values?.[length - 1]?.id;
+    if (!values) return undefined;
+
+    for (const attestation of values) {
+      if (config.admins.indexOf(attestation.attester) !== -1) {
+        return attestation.id;
+      }
+    }
+    return undefined;
```

Comment:
Don't use indexOf and `for of`.
Instead of `indexOf` use `includes` and instead of `for of` use `find`